src/lib/merkle_ledger_tests/test_mask.ml

```diff
@@ -156,477 +172,505 @@ module Make (Test : Test_intf) = struct
     in
     test_hashes_at_address addr
 
-  let%test "parent, mask agree on set" =
```

Comment:
Do I understand correctly that this test was written twice (same code, duplicated)?

Reply:
Exactly